### DISCUSSION TITLE
altlantic scinetific whitebox t3 for multiple sensors?

### DISCUSSION THREAD

**Discussion by Cinett87** [theyosh/TerrariumPI#743] [Q&A] [answered]
the link to "see more info" takes me directly to their home page, while digging around I found this product which does say its 12c connections only on it but is it terrarium compatible as a hub? https://atlas-scientific.com/electrical-isolation/whitebox-t3/

**Comment by theyosh** [comment, marked as answer]
Hi, see at https://github.com/theyosh/TerrariumPI/issues/729#issuecomment-1279744414
